### PR TITLE
Add longevity pipelines to extend GCP coverage

### PR DESCRIPTION
**Pull request by Mark-Gurevich**
### This is the first PR for extending GCE coverage

Based on task: https://github.com/scylladb/qa-tasks/issues/1134

### Add pipelines in this PR:

1. longevity-200gb-48h-gce.jenkinsfile
2. longevity-150gb-asymmetric-cluster-12h-gce.jenkinsfile
3. longevity-large-partition-8h-gce.jenkinsfile
4. longevity-large-partition-200k-pks-2days-aws-i4i-gce.jenkinsfile
5. longevity-alternator-200gb-48h-gce.jenkinsfile
6. longevity-lwt-24h-gce.jenkinsfile
7. longevity-multi-keyspaces-60h-gce.jenkinsfile (Use even 64 cores)
8. longevity-topology-changes-3h-gce.jenkinsfile (Duplicate it and extend it to run 24h also for aws)
9. longevity-topology-changes-24h-gce.jenkinsfile 
10. longevity-mv-si-4days-gce.jenkinsfile (Duplicate it and decrease it to run 24h)
11. longevity-mv-si-24h-gce.jenkinsfile

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [ ] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [ ] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

File: test-cases/longevity/longevity-150GB-12h-autorization-LimitedMonkey.yaml
Comment (fgelcer) [inline test-cases/longevity/longevity-150GB-12h-autorization-LimitedMonkey.yaml:12]: `i3.4xlarge` has 16 vCPUs
shouldn't we be fitting the sizes?

File: test-cases/longevity/longevity-200GB-48h-verifier-LimitedMonkey-tls.yaml
Comment (fgelcer) [inline test-cases/longevity/longevity-200GB-48h-verifier-LimitedMonkey-tls.yaml:14]: same here

File: test-cases/longevity/longevity-alternator-200GB-48h.yaml
Comment (fgelcer) [inline test-cases/longevity/longevity-alternator-200GB-48h.yaml:41]: same here

File: test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml
Comment (fgelcer) [inline test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml:62]: `i3en.3xlarge` has 12 vCPUs... here you are adding 32

File: test-cases/longevity/longevity-large-partition-8h.yaml
Comment (fgelcer) [inline test-cases/longevity/longevity-large-partition-8h.yaml:45]: here we have 8 vCPUs, you are adding 32

File: test-cases/longevity/longevity-lwt-basic-24h.yaml
Comment (fgelcer) [inline test-cases/longevity/longevity-lwt-basic-24h.yaml:14]: same here

File: test-cases/longevity/longevity-mv-si-24h.yaml
Comment (fgelcer) [inline test-cases/longevity/longevity-mv-si-24h.yaml:19]: same here

File: test-cases/longevity/longevity-mv-si-4days.yaml
Comment (fgelcer) [inline test-cases/longevity/longevity-mv-si-4days.yaml:18]: same here

File: test-cases/longevity/longevity-topology-changes-24h.yaml
Comment (fgelcer) [inline test-cases/longevity/longevity-topology-changes-24h.yaml:22]: here, the GCE instance has a bit less memory than the AWS peer.. we should test this one too, before merging, to be sure it will be enough

File: test-cases/longevity/longevity-topology-changes-3h.yaml
Comment (fgelcer) [inline test-cases/longevity/longevity-topology-changes-3h.yaml:21]: same here.. but perhaps testing the 24h first, and if that one passes, we won't need to test this one

File: test-cases/longevity/longevity-150GB-12h-autorization-LimitedMonkey.yaml
Comment (Mark-Gurevich) [reply test-cases/longevity/longevity-150GB-12h-autorization-LimitedMonkey.yaml:12]: We wanted to test 24/32 cores for large instances that's why I put 32 here

File: test-cases/longevity/longevity-200GB-48h-verifier-LimitedMonkey-tls.yaml
Comment (Mark-Gurevich) [reply test-cases/longevity/longevity-200GB-48h-verifier-LimitedMonkey-tls.yaml:14]: We wanted to test 24/32 cores for large instances that's why I put 32 here

File: test-cases/longevity/longevity-150GB-12h-autorization-LimitedMonkey.yaml
Comment (fgelcer) [reply test-cases/longevity/longevity-150GB-12h-autorization-LimitedMonkey.yaml:12]: was it necessary for all tests? or just some specific ones?

File: test-cases/longevity/longevity-topology-changes-3h.yaml
Comment (Mark-Gurevich) [reply test-cases/longevity/longevity-topology-changes-3h.yaml:21]: For 24 we need to create custom instance type. There are 16 or 32 instances in gce (screenshots from gce website in the task). We were asked to test large instances that's why I put 32 cores here as well. 
If we want only a few pipelines with large instances with 32 cores please specify which pipelines to use with 32 and which with 16/8.

File: test-cases/longevity/longevity-150GB-12h-autorization-LimitedMonkey.yaml
Comment (Mark-Gurevich) [reply test-cases/longevity/longevity-150GB-12h-autorization-LimitedMonkey.yaml:12]: Changed to 16

File: test-cases/longevity/longevity-200GB-48h-verifier-LimitedMonkey-tls.yaml
Comment (Mark-Gurevich) [reply test-cases/longevity/longevity-200GB-48h-verifier-LimitedMonkey-tls.yaml:14]: Changed to 16

File: test-cases/longevity/longevity-alternator-200GB-48h.yaml
Comment (Mark-Gurevich) [reply test-cases/longevity/longevity-alternator-200GB-48h.yaml:41]: left 32

File: test-cases/longevity/longevity-large-partition-8h.yaml
Comment (Mark-Gurevich) [reply test-cases/longevity/longevity-large-partition-8h.yaml:45]: Changed to 8

File: test-cases/longevity/longevity-lwt-basic-24h.yaml
Comment (Mark-Gurevich) [reply test-cases/longevity/longevity-lwt-basic-24h.yaml:14]: Changed to 8

File: test-cases/longevity/longevity-mv-si-24h.yaml
Comment (Mark-Gurevich) [reply test-cases/longevity/longevity-mv-si-24h.yaml:19]: left 32

File: test-cases/longevity/longevity-mv-si-4days.yaml
Comment (Mark-Gurevich) [reply test-cases/longevity/longevity-mv-si-4days.yaml:18]: left 32

File: test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml
Comment (aleksbykov) [inline test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml:62]: does n1-highmem-16 is equal by parameters to i3en.3xlarge?

File: test-cases/longevity/longevity-mv-si-4days.yaml
Comment (aleksbykov) [reply test-cases/longevity/longevity-mv-si-4days.yaml:18]: above you use n1-highmem-16 for i3.4xlage, why here n1-32?

File: test-cases/longevity/longevity-topology-changes-3h.yaml
Comment (aleksbykov) [reply test-cases/longevity/longevity-topology-changes-3h.yaml:22]: Please , also check the memory and HDD

**Review by aleksbykov — COMMENTED** [review]
See my comments.

File: test-cases/longevity/longevity-mv-si-4days.yaml
Comment (Mark-Gurevich) [reply test-cases/longevity/longevity-mv-si-4days.yaml:18]: Because @roydahan asked in the task to add tests with many cores 24/32, so I chose "randomly" some tests where the instance is with 32 cores.

**Comment by fgelcer** [comment]
ping @roydahan

File: jenkins-pipelines/longevity-alternator-200gb-48h-gce.jenkinsfile
Comment (roydahan) [inline jenkins-pipelines/longevity-alternator-200gb-48h-gce.jenkinsfile:12]: Why this one specify it?
Please remove, it should be the default.

File: test-cases/longevity/longevity-150GB-12h-autorization-LimitedMonkey.yaml
Comment (roydahan) [inline test-cases/longevity/longevity-150GB-12h-autorization-LimitedMonkey.yaml:31]: @fgelcer please open a task to remove this from here, put it in the default for gce_config and add "false" to all places we need to specify it like artifacts and rolling upgrades.

File: test-cases/longevity/longevity-150GB-12h-autorization-LimitedMonkey.yaml
Comment (roydahan) [inline test-cases/longevity/longevity-150GB-12h-autorization-LimitedMonkey.yaml:12]: Please change all to n2.

File: test-cases/longevity/longevity-5TB-1day-gce.yaml
Comment (roydahan) [inline test-cases/longevity/longevity-5TB-1day-gce.yaml:25]: Change to 24, 16 is not enough for 5TB.

File: test-cases/longevity/longevity-5TB-1day-gce.yaml
Comment (roydahan) [inline test-cases/longevity/longevity-5TB-1day-gce.yaml:5]: This is preparing 1TB, not 5TB.

File: test-cases/longevity/longevity-5TB-1day-gce.yaml
Comment (roydahan) [inline test-cases/longevity/longevity-5TB-1day-gce.yaml:8]: This is mixed that goes over 1TB...

File: test-cases/longevity/longevity-5TB-1day-gce.yaml
Comment (roydahan) [reply test-cases/longevity/longevity-5TB-1day-gce.yaml:5]: just noticed it's using 5kb rows.

File: test-cases/longevity/longevity-5TB-1day-gce.yaml
Comment (roydahan) [inline test-cases/longevity/longevity-5TB-1day-gce.yaml:9]: Please change to use profile:
mv_synchronous_updates_on_standard.yaml

File: test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml
Comment (Mark-Gurevich) [reply test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml:62]: It is slightly bigger and faster; should be fine. There is no identical match for `i3en.3xlarge` in "GCP"

File: jenkins-pipelines/longevity-topology-changes-3h-gce.jenkinsfile
Comment (Mark-Gurevich) [reply jenkins-pipelines/longevity-topology-changes-3h-gce.jenkinsfile:13]: Done

File: test-cases/longevity/longevity-5TB-1day-gce.yaml
Comment (roydahan) [inline test-cases/longevity/longevity-5TB-1day-gce.yaml:38]: please change to 5tb and remove the 7d.

File: test-cases/longevity/longevity-multi-keyspaces.yaml
Comment (roydahan) [inline test-cases/longevity/longevity-multi-keyspaces.yaml:25]: gce_n_local_ssd_disk_db: 16

File: test-cases/longevity/longevity-lwt-basic-24h.yaml
Comment (roydahan) [inline test-cases/longevity/longevity-lwt-basic-24h.yaml:14]: gce_n_local_ssd_disk_db: 8

File: test-cases/longevity/longevity-topology-changes-3h.yaml
Comment (Mark-Gurevich) [reply test-cases/longevity/longevity-topology-changes-3h.yaml]: Changed all 'n1's to 'n2's.
n2-highmem-32 has the same memory as i4i.8xlarge. 
We don't need a special run nor for this one neither the 24h.

File: test-cases/longevity/longevity-topology-changes-24h.yaml
Comment (Mark-Gurevich) [reply test-cases/longevity/longevity-topology-changes-24h.yaml]: No need as changed to n2-highmem-32

File: test-cases/longevity/longevity-alternator-200GB-48h.yaml
Comment (roydahan) [inline test-cases/longevity/longevity-alternator-200GB-48h.yaml:42]: gce_n_local_ssd_disk_db: 16

File: test-cases/longevity/longevity-mv-si-24h.yaml
Comment (roydahan) [inline test-cases/longevity/longevity-mv-si-24h.yaml:1]: We don't need this one.
The one of mv-si-4days is good enough (we have now generic test duration).

File: jenkins-pipelines/longevity-mv-si-24h-gce.jenkinsfile
Comment (roydahan) [inline jenkins-pipelines/longevity-mv-si-24h-gce.jenkinsfile:1]: We don't need this one.

File: test-cases/longevity/longevity-mv-si-4days.yaml
Comment (roydahan) [inline test-cases/longevity/longevity-mv-si-4days.yaml:19]: gce_n_local_ssd_disk_db: 16

File: test-cases/longevity/longevity-topology-changes-24h.yaml
Comment (roydahan) [inline test-cases/longevity/longevity-topology-changes-24h.yaml:1]: I see it's a completely new test, was it tested?

Please change the name to:
test-cases/longevity/longevity-topology-changes-multi-AZ-asymetric-shards-24h.yaml

File: test-cases/longevity/longevity-topology-changes-24h.yaml
Comment (roydahan) [inline test-cases/longevity/longevity-topology-changes-24h.yaml:16]: Do we even have SCT support for multi AZ in GCE?

@soyacz ?

File: test-cases/longevity/longevity-topology-changes-24h.yaml
Comment (roydahan) [inline test-cases/longevity/longevity-topology-changes-24h.yaml:6]:
```suggestion
                     "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=1250 -clustering-row-count=10000 -clustering-row-size=uniform:1024..8192 -concurrency=25 -connection-count=25 -consistency-level=quorum -rows-per-request=10 -timeout=30s -retry-number=30 -retry-interval=80ms,1s"
```

File: test-cases/longevity/longevity-topology-changes-24h.yaml
Comment (roydahan) [inline test-cases/longevity/longevity-topology-changes-24h.yaml:4]:
```suggestion
pre_create_keyspace: "CREATE KEYSPACE scylla_bench WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': '3'};"
```

Comment (roydahan) [reply test-cases/longevity/longevity-topology-changes-24h.yaml:4]: Same for all commands below

File: test-cases/longevity/longevity-topology-changes-24h.yaml
Comment (roydahan) [inline test-cases/longevity/longevity-topology-changes-24h.yaml:18]:
```suggestion
n_loaders: 4
```

File: test-cases/longevity/longevity-topology-changes-24h.yaml
Comment (roydahan) [inline test-cases/longevity/longevity-topology-changes-24h.yaml:17]:
```suggestion
n_db_nodes: 6
```

File: test-cases/longevity/longevity-topology-changes-24h.yaml
Comment (roydahan) [inline test-cases/longevity/longevity-topology-changes-24h.yaml:32]:
```suggestion
seeds_num: 6
```

File: test-cases/longevity/longevity-topology-changes-24h.yaml
Comment (roydahan) [inline test-cases/longevity/longevity-topology-changes-24h.yaml:12]:
```suggestion
             "scylla-bench -workload=sequential -mode=write -replication-factor=3 -partition-count=1000 -clustering-row-count=10000 -clustering-row-size=uniform:1024..8192 -concurrency=25 -connection-count=25 -consistency-level=quorum -rows-per-request=10 -timeout=30s -retry-number=30 -retry-interval=80ms,1s -iterations 0 -duration=1440m",
```

Comment (roydahan) [reply test-cases/longevity/longevity-topology-changes-24h.yaml:12]: We can leave 250 partitions to be used for "validate_partitions".
@yarongilor please help to define the set of needed params.

Comment (roydahan) [reply test-cases/longevity/longevity-topology-changes-24h.yaml:12]: @Mark-Gurevich on a second thought, please take this test out of this PR to its own PR.

File: test-cases/longevity/longevity-topology-changes-24h.yaml
Comment (roydahan) [inline test-cases/longevity/longevity-topology-changes-24h.yaml:13]:
```suggestion
             "scylla-bench -workload=sequential -mode=read  -replication-factor=3 -partition-count=1250 -clustering-row-count=10000 -clustering-row-size=uniform:1024..8192 -concurrency=25 -connection-count=25 -consistency-level=quorum -rows-per-request=10 -timeout=30s -retry-number=30 -retry-interval=80ms,1s -iterations 0 -duration=1440m",
```

File: test-cases/scale/scale-cluster.yaml
Comment (roydahan) [inline test-cases/scale/scale-cluster.yaml:1]: remove this one, I'm refactoring it in a different PR.

File: test-cases/longevity/longevity-mv-si-24h.yaml
Comment (Mark-Gurevich) [reply test-cases/longevity/longevity-mv-si-24h.yaml:1]: Sorry for "stupid" question but can I delete this file from the repo or just reverse changes?

File: test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml
Comment (Mark-Gurevich) [reply test-cases/longevity/longevity-large-partition-200k_pks-4days.yaml:66]: Done

File: test-cases/longevity/longevity-alternator-200GB-48h.yaml
Comment (Mark-Gurevich) [reply test-cases/longevity/longevity-alternator-200GB-48h.yaml:43]: done

File: test-cases/longevity/longevity-lwt-basic-24h.yaml
Comment (Mark-Gurevich) [reply test-cases/longevity/longevity-lwt-basic-24h.yaml:14]: done

File: test-cases/longevity/longevity-mv-si-24h.yaml
Comment (Mark-Gurevich) [reply test-cases/longevity/longevity-mv-si-24h.yaml:1]: deleted

File: test-cases/longevity/longevity-multi-keyspaces.yaml
Comment (Mark-Gurevich) [reply test-cases/longevity/longevity-multi-keyspaces.yaml:26]: done

**Review by roydahan — DISMISSED** [review]

**Comment by roydahan** [comment]
@fgelcer please review and merge if you find it good.

File: test-cases/longevity/longevity-150GB-12h-autorization-LimitedMonkey.yaml
Comment (fgelcer) [reply test-cases/longevity/longevity-150GB-12h-autorization-LimitedMonkey.yaml:31]: https://github.com/scylladb/qa-tasks/issues/1371

File: test-cases/longevity/longevity-5TB-1day-gce.yaml
Comment (fgelcer) [inline test-cases/longevity/longevity-5TB-1day-gce.yaml:1]: this file's name should be without GCE, as it is kind of default to, like all other `*.yaml` your are changing in this PR

**Review by fgelcer — COMMENTED** [review]
little comment, but other than that, LGTM

**Review by fgelcer — APPROVED** [review]
LGTM

**Review by aleksbykov — APPROVED** [review]
LGTM

**Comment by fgelcer** [comment]
@scylladb/qa-maintainers , can we please backport this one to `branch-2023.1`?

**Pull request merged**
